metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Installs/Configures auditd'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.3.4'
+version '2.3.5'
```

Comment:
We have tooling that handles the version bump and changelog generation so you can leave this out in the future.